### PR TITLE
Make new site-changes page

### PR DESCRIPTION
**Pull request by nicholsj**
Making a new page which we can redirect various urls to (eg forum/apps/commenting etc) - we won't link to this page from anywhere else. 

Content for the page will come later. 

https://trello.com/c/HUBHaeoJ/227-page-for-retired-functionality

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -16,6 +16,9 @@ def support
   def accessibility
   end
 
+  def site-changes
```
Comment (andrewgarner) [inline app/controllers/pages_controller.rb]: The convention is to use underscores, so this should be `site_changes`.

File: app/views/pages/site-changes.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<% content_for :page_title do %>
```
Comment (andrewgarner) [inline app/views/pages/site-changes.html.erb]: The path to this file should be `app/views/pages/site_changes.html.erb`

File: config/routes.rb [hunk 3]
```diff
@@ -21,6 +21,7 @@
     get 'support'
     get 'terms'
     get 'dashboard'
+    get 'site-changes'
```
Comment (andrewgarner) [inline config/routes.rb]: `get 'site-changes', to: :site_changes` should allow you to maintain the desired path but use underscoring for the controller method.

**Comment by nicholsj** [comment]
@andrewgarner I've made those changes

File: config/routes.rb [hunk 4]
```diff
@@ -21,7 +21,7 @@
     get 'support'
     get 'terms'
     get 'dashboard'
-    get 'site-changes'
+    get 'site_changes'
```
Comment (andrewgarner) [inline config/routes.rb]: If you change this to `get 'site-changes', to: :site_changes` you can have the URL "/site-changes" which I believe is preferable.

**Comment by nicholsj** [comment]
Thanks @andrewgarner, change made

**Pull request merged**